### PR TITLE
Signature 'Garrard Kitchen'

### PR DESCRIPTION
**Pull request by garrardkitchen**
# Description

Added name to list of signatures

## Type of change

- [x] Signature (please update title to "Signature 'your name'")

### CODE REVIEW

File: content/MinimumCD/signatures.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ weight: 2
 Do you want to become a signatory? Read our [contribution guidelines](https://github.com/Minimum-CD/cd-manifesto/blob/master/CONTRIBUTING.md).
 
 {{%excerpt%}}
-
+[Garrard&nbsp;Kitchen](https://www.linkedin.com/in/garrard-kitchen/)&nbsp;-
```
Comment (ibathazi) [inline content/MinimumCD/signatures.md:10]: Please move signature to the end of the file

**Review by ibathazi — CHANGES_REQUESTED** [review]
Please move signature to the end of the file

File: content/MinimumCD/signatures.md (hunk 1 above)
Comment (garrardkitchen) [reply content/MinimumCD/signatures.md:10]: done

**Review by bdfinst — APPROVED** [review]
Sorry for the delay. We had a broken build on Netlify.

**Pull request merged**